File: cycle/mysql_test_reader.py
#!/usr/bin/python

import peewee
from peewee import *
import sys

MYSQL_DB = "scale_cycle"
MYSQL_USER = "scale_usr"
MYSQL_PASSWD = "123456"

class EventRecord(peewee.Model):
	class Meta:
		database = None
		db_table = "events"

	sensor = peewee.CharField(max_length = 32)
	msg = peewee.TextField()
	priority = peewee.IntegerField()
	timestamp = peewee.DoubleField()
	gpstamp = peewee.TextField(null = True)
	upldstamp = peewee.DoubleField(null = True)

db = MySQLDatabase(MYSQL_DB, user = MYSQL_USER, passwd = MYSQL_PASSWD)
try:
	db.connect()
except OperationalError, err:
	print err
	exit(-1)

EventRecord._meta.database = db
res = EventRecord.select()
count = res.wrapped_count()
print "Fetched %d record(s)." % count
db.close()

File: cycle/mysql_test_reader.php
<?php
$MYSQL_HOST = "localhost";
$MYSQL_DB = "scale_cycle";
$MYSQL_USER = "scale_usr";
$MYSQL_PASSWD = "123456";
$DB_TABLE = "events";

if(!mysql_connect($MYSQL_HOST, $MYSQL_USER, $MYSQL_PASSWD))
	die("Cannot access database server.");
if(!mysql_select_db($MYSQL_DB))
	die("Cannot select database.");

$res = mysql_query("SELECT * FROM {$DB_TABLE}");
if(!$res)
	die("Cannot access table.");
$count = mysql_num_rows($res);
print("Fetched {$count} record(s).".PHP_EOL);
?>